src/domain/Space.ts

```diff
@@ -0,0 +1,43 @@
+type SpaceType = "CAMPUS" | "BUILDING" | "FLOOR" | "ROOM" | "ROOM_SUBDIVISION"
+export type RoomInfo = { id: string, name: string}
+
+export type Space = {
+	id: string
+	name: string
+	fullName: string
+	type: SpaceType
+	classification: Record<string, string>
+	capacity: {
+		regular: number
+		exam: number
+	}
+	description: string
+	building?: {	
+		id: string
+		name: string
+		fullName: string
+		type: SpaceType
+		classification: Record<string, string>
+	}
+	campus?: {
+		id: string		
+		name: string
+		fullName: string
+		type: SpaceType
+		classification: Record<string, string>
+	}
+	containedIn: {
+		id: string	
+		name: string
+		fullName: string
+		type: SpaceType
+		classification: Record<string, string>
+	}
+	contains: {
+		id: string		
+		name: string
+		fullName: string
+		type: SpaceType
+		classification: Record<string, string>
+	}[]
+}
```

Comment:
Missing newline at the end of the file, I think this is recurrent but we never set up the linter/formatter